weave-js/src/components/Button/Button.tsx

```diff
@@ -156,14 +156,26 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     );
 
     const [isTooltipOpen, setIsTooltipOpen] = useState(false);
+    const handleMouseEnter = () => {
+      setIsTooltipOpen(true);
+    };
+
+    const handleMouseLeave = () => {
+      setIsTooltipOpen(false);
+    };
+
     if (tooltip) {
       return (
         <Tailwind style={wrapperStyles}>
           <Tooltip.Provider>
             <Tooltip.Root open={isTooltipOpen} onOpenChange={setIsTooltipOpen}>
               <Tooltip.Trigger asChild>
-                {/* span is needed so tooltip works on disabled buttons */}
-                <span className="[display:inherit]">{button}</span>
+                <span
+                  onMouseEnter={handleMouseEnter}
+                  onMouseLeave={handleMouseLeave}>
```

Comment:
hmm this change doesn't seem right. there's probably something wrong with the call site where it's not working.

Reply:
Sounds good -- investigating more